.gitignore

```diff
@@ -1,3 +1,2 @@
 /vendor
-composer.lock
-composer.phar
\ No newline at end of file
+composer.phar
```

Comment:
`composer.lock` should also be ignore and never commit as it's a third-party library.